Pid.hpp

```diff
@@ -31,4 +42,6 @@ class Pid {
     float _lastErr;
 
     std::vector<float> _oldErr;
+
+    pidTuner* _tuner;
```

Comment:
Could you change this to std::unique_ptr<pidTuner> _tuner.
And change the initialize function to 
_tuner = make_unique<pidTuner>(kp,ki,kd);
It'll prevent memory leaks.